### ISSUE TITLE
atom-text-editor.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by dieferman** [thomaslindstrom/color-picker#199]
The contents of `atom-text-editor` elements are no longer encapsulated
within a shadow DOM boundary. Please, stop using `shadowRoot` and access
the editor contents directly instead.
```
atom-text-editor.Object.defineProperty.get (C:\Users\diefer\AppData\Local\atom\app-1.13.0-beta2\resources\app.asar\src\text-editor-element.js:58:18)
Object.open (C:\Users\diefer\.atom\packages\color-picker\lib\ColorPicker-view.coffee:318:32)
atom-text-editor.<anonymous> (C:\Users\diefer\.atom\packages\color-picker\lib\ColorPicker.coffee:36:29)
CommandRegistry.handleCommandEvent (C:\Users\diefer\AppData\Local\atom\app-1.13.0-beta2\resources\app.asar\src\command-registry.js:259:29)
handleCommandEvent (C:\Users\diefer\AppData\Local\atom\app-1.13.0-beta2\resources\app.asar\src\command-registry.js:3:59)
CommandRegistry.dispatch (C:\Users\diefer\AppData\Local\atom\app-1.13.0-beta2\resources\app.asar\src\command-registry.js:160:19)
```

**Comment by naymaeu** [comment]
I have this same kind of issue.

**Comment by srhnyldz** [comment]
I have the same issue

**Comment by jadchaar** [comment]
Same issue as Atom 1.13 has reached the stable channel

**Comment by HUN73R13** [comment]
same issue

**Comment by HUN73R13** [comment]
I was able to "solve" the issue by editing **ColorPicker-view.cofee**
just find `EditorRoot = EditorView.shadowRoot or EditorView` on line 265 and comment it out and put a new line without _EditorView.shadowRoot_ so you have

```
# EditorRoot = EditorView.shadowRoot or EditorView
EditorRoot = EditorView
```

I'm in no way sure about the validity of this solution but it seems to work for me and I no longer get the deprecation message

**Comment by thomaslindstrom** [comment]
Hi, guys. This should be fixed in `v2.2.5`.
